File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1099.lua
n = io.read("*number")
i=0
while i<n do
    x = io.read("*number")
    y = io.read("*number")
    si=0
    if(x>y) then
        temp=x
        x=y
        y=temp
    end
    if(x%2==0) then
        x=x+1
    else
        x=x+2
    end
    j=x
    while j<y do
        si=si+j
        j=j+2
    end
    print(si)
    i=i+1
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1099.cpp
#include <iostream>

using namespace std;

int main(){
    int n, x, y, si, temp;
    cin >> n;
    for(int i=0;i<n;i++){
        cin >> x >> y;
        si=0;
        if(x>y){
            temp=x;
            x=y;
            y=temp;
        }

        if(x%2==0){
            x++;
        } else{
            x+=2;
        }
        for(int j=x;j<y;j+=2){
            si+=j;
        }
        cout << si << endl;
    }

    return 0;
}